lib/ASTGen/Sources/ASTGen/Bridge.swift

```diff
@@ -290,3 +310,11 @@ extension ConcatCollection: LazyCollectionProtocol {
     }
   }
 }
+
+extension BridgedArrayRef {
+  public func withElements<T, R>(ofType ty: T.Type, _ c: (UnsafeBufferPointer<T>) -> R) -> R {
+    let start = data?.bindMemory(to: ty, capacity: count)
```

Comment:
I feel like this should probably either use `withMemoryRebound(to:)` or `assumingMemoryBound(to:)`

Reply:
Indeed, I'd go with `assumingMemoryBound`. I copied this from SwiftCompilerSources, so I will change both in followups.